### PR TITLE
Adding k8s-netperf in grafonnet

### PR DESCRIPTION
**Pull request by vishnuchalla**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [x] Optimization
- [ ] Documentation Update

## Description
Upstream code change for missing bits: https://github.com/grafana/grafonnet/pull/140
Adding k8s-netperf dashboard in grafonnet.

## Related Tickets & Documents

- Related Issue - https://github.com/cloud-bulldozer/performance-dashboards/issues/77

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
Tested using v2 instructions in Makefile.

### CODE REVIEW

**Comment by vishnuchalla** [comment]
@jtaleric Requesting a review on these updated changes. We do have everything in the new grafonnet library except for these two attributes: https://github.com/grafana/grafonnet/pull/140/files that I am trying to get added in the upstream schema.

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ mkdir -p rendered/General/
 bin/jsonnet -J ./templates/vendor templates/General/ocp-performance-v2.jsonnet > rendered/General/ocp-performance-v2.json
 Rendered the v2 dashboards with latest grafonnet library
 ```
-All other operations remain same as before.
+Rest all operations reamin same as before.
```
Comment (jtaleric) [inline README.md:58]: Not following this line. also, remain.

Comment (vishnuchalla) [reply README.md]: Its just an update from current master branch. Corrected the typo.

**Comment by vishnuchalla** [comment]
> lgtm for the current dashboard.
> 
> I would love to see something similar to what we have with ingress-perf but for k8s-netperf.

Can you elaborate on the specifics please, so that I can make note of it and get it added in the upcoming PRs.

**Pull request merged**